server/chat-plugins/scavenger-games.ts

```diff
@@ -364,16 +364,16 @@ const TWISTS: {[k: string]: Twist} = {
 		},
 	},
 
-	mistakes: {
-		id: 'mistakes',
-		name: 'Mistakes',
+	scavengersfeud: {
+		id: 'scavengersfeud',
+		name: 'Scavengers Feud',
 		desc: 'After completing the hunt, players will guess what the most common incorrect answer for each question is.',
 		onAfterLoad() {
 			this.guesses = {};
-			this.incorrect = new Array(this.questions.length).fill({});
+			this.incorrect = new Array(this.questions.length).fill('').map(() => ({}));
```

Comment:
```suggestion
			this.incorrect = new Array(this.questions.length).map(() => ({}));
```
One less extra loop.

Reply:
apparently node is stupid and this doesn't work nvm